src/modm/platform/can/stm32/can.cpp.in

```diff
@@ -201,6 +201,7 @@ readMailbox(modm::can::Message& message, uint32_t mailboxId)
 	message.setRemoteTransmitRequest(rir & CAN_RI0R_RTR);
 
 	message.length = mailbox->RDTR & CAN_TDT1R_DLC;
+    message.filter_id = (mailbox->RDTR & CAN_RDT1R_FMI) >> CAN_RDT1R_FMI_Pos;
```

Comment:
The `getMessage` function might be a better place to return this value. E.g. as an optional reference parameter.